.zuul.yaml

```diff
@@ -72,15 +72,18 @@
               - ^pkg/util/.*
               - ^go.mod$
               - ^go.sum$
+    cloud-provider-openstack-acceptance-test-flexvolume-cinder:
+      jobs:
+        - cloud-provider-openstack-acceptance-test-flexvolume-cinder:
             irrelevant-files:
               - ^docs/.*$
               - ^.*\.md$
               - ^OWNERS$
               - ^SECURITY_CONTACTS$
               - ^.gitignore$
-    cloud-provider-openstack-acceptance-test-flexvolume-cinder:
+    cloud-provider-openstack-acceptance-test-csi-manila:
       jobs:
-        - cloud-provider-openstack-acceptance-test-flexvolume-cinder:
+        - cloud-provider-openstack-acceptance-test-csi-manila:
```

Comment:
could you update it to run only for csi manila changes ?

Reply:
So can we make this:
    cloud-provider-openstack-acceptance-test-flexvolume-cinder:
      jobs:
        - cloud-provider-openstack-acceptance-test-flexvolume-cinder:
            files:
              - ^cmd/manila-csi-plugin/.*
              - ^examples/manila-csi-plugin/.*
              - ^manifests/manila-csi-plugin/.*
              - ^pkg/csi/manila/.*
              - ^pkg/util/.*
              - ^go.mod$
              - ^go.sum$
            irrelevant-files:
               ... (as before)

Reply:
yes , 2 things -
1. job name would be cloud-provider-openstack-acceptance-test-csi-manila
2. looks like only pkg/util/errors is used in manila csi , we could restrict only to that

Reply:
Thanks for your feedback, I've addressed your comments in https://github.com/kubernetes/cloud-provider-openstack/pull/843/commits/afab2b8495041da71a90a17fa3dfd20e1493a00b . We'll have to be careful and add/remove dependencies to the white-list as they come and go in the future.

Reply:
+1

Reply:
> yes , 3 things -
> 
>     1. job name would be cloud-provider-openstack-acceptance-test-csi-manila
> 
>     2. looks like only pkg/util/errors is used in manila csi , we could restrict only to that
> 
>     3. job doesn't seem to use manifests/manila-csi-plugin for deployment , so no need to add that too

Thanks @ramineni, and sorry about the copy-paste error on the job name!